openssh.yaml

```diff
@@ -121,12 +121,17 @@ subpackages:
     description: "OpenSSH server"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/sbin
+          mkdir -p "${{targets.subpkgdir}}"/usr/sbin "${{targets.subpkgdir}}"/var/empty
+          chmod 755 "${{targets.subpkgdir}}"/var/empty
           mv "${{targets.destdir}}"/usr/sbin/sshd "${{targets.subpkgdir}}"/usr/sbin/
     dependencies:
       runtime:
         - openssh-keygen
         - openssh-server-config
+    scriptlets:
+      post-install: |
+        #!/bin/sh
+        ssh-keygen -A -q -N ""
```

Comment:
I'm skeptical of scriptlets in general, since they're not executed by apko, only on `apk add`. Is there a way to get this side effect for apko-built users too?

Reply:
Yep.  Teach apko to run scriptlets.

Reply:
looks like this is failing in presubmit tests:

```
2024/06/17 15:04:57 INFO running the main test pipeline
2024/06/17 15:04:57 WARN + export 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
2024/06/17 15:04:57 WARN + '[' -d /home/build ]
2024/06/17 15:04:57 WARN + cd /home/build
2024/06/17 15:04:57 WARN + /usr/sbin/sshd
2024/06/17 15:04:57 WARN sshd: no hostkeys available -- exiting.
```

Likely becauase we use apko to build the test workspace, which doesn't yet know about scriptlets

Reply:
Well....should I force the keygen in the test itself to make it pass?

Reply:
I just pushed a change that forces the ssh key gen in the test itself, with a comment

Reply:
> Yep. Teach apko to run scriptlets.

That makes no sense here =) because we do not want to bake the same host key into the golden image at apko execution time.

We would only want to do this at image startup / runtime, and ideally provision onto a persistent volume.